.github/workflows/android-appcenter.yml

```diff
@@ -6,16 +6,14 @@ on:
     branches: [ master ]
     paths:
       - '.github/workflows/android-appcenter.yml'
-      - 'demo/android/Activity/rhino-activity-demo-app/**'
-      - '!demo/android/Activity/rhino-activity-demo-app/README.md'
+      - 'demo/android/Activity/**'
       - 'resources/.test/**'
       - 'resources/audio_samples/**'
   pull_request:
     branches: [ master, 'v[0-9]+.[0-9]+' ]
     paths:
       - '.github/workflows/android-appcenter.yml'
-      - 'demo/android/Activity/rhino-activity-demo-app/**'
-      - '!demo/android/Activity/rhino-activity-demo-app/README.md'
+      - 'demo/android/Activity/**'
```

Comment:
`- '!demo/android/Activity/README.md'`

Reply:
Since there is only one demo in the Rhino repo the README is actually located outside of `Activity`